packages/docz-tools/README.md

```diff
@@ -123,14 +123,16 @@ export default {
 
 ### Hide / Show Logo
 
-The logo within the sidebar can be hidden if required by providing an
-`undefined` value to `title`.
+The logo within the sidebar can be hidden if required by adjusting `hasLogo` to
+false. By default it is `true`
 
 ```
 // doczrc.js
 export default {
   ...,
-  title: undefined
+  themeConfig: {
+  hasLogo: false
+  }
```

Comment:
Can we add the spread `...` and fix indentation on this?
```
themeConfig: {
  ....
   hasLogo: false
}

Reply:
added spread, fixed indentation in e1e67d5